src/plugin.js

```diff
@@ -187,6 +187,7 @@ class TropiiifyPlugin {
       await new Promise((resolve, reject) => {
         sharpInstance
           .resize(maxDimension, maxDimension, { fit: 'inside' })
+          .jpeg()
```

Comment:
Hi @atomotic, you will need to change the destination path in line 205 as well so the file gets saved with the correct extension